.buildkite/lint.rayci.yml

```diff
@@ -73,4 +73,3 @@ steps:
       - pip install -r ci/compile_py_proto/requirements_compile_py_proto.txt
       - ./ci/lint/check-python-proto.sh
     depends_on: forge
-
```

Comment:
why is this file change required?

Reply:
I don't think it's required, probably bc my ide was setup to drop extra time in the file. I can but the extra line back